script/deploy-canary-demo

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -euxo pipefail
+version=${1+$1}
```

Comment:
Kinda like the infix or suffix name earlier. This prev-major is not a version :p.
Also I never remember any bash syntax but I assume this is a just a fancy way of saying if $1 is set then version = $1? Is it any different from just version = $1?

Reply:
How about `canarySufix`? Also it is effectively version=$1, but the main difference is that when I tested giving no param to the script with version=$1 it gave a error because 1 was never set, but this syntax is safer because it effectively checks if $1 exists before setting it to $1, effectively equates to this:
`
if($1)
  version=$1
else
  version=nothing
`

Reply:
`canarySuffix` looks good to me.

Reply:
oh right, could you also add a comment for this line?
> if($1) version=$1 else version=nothing
this is good enough

Reply:
Yeah canarySuffix sounds good enough

Reply:
Updated, although deploying for previous version won't work until our s3 bucket limit is increased, but based on output there isn't anything wrong with the code itself and we can merge. https://github.com/aws/amazon-chime-sdk-js/runs/5073025117?check_suite_focus=true